test_package/CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 project(PackageTest CXX)
 cmake_minimum_required(VERSION 3.0)
-add_compile_options(-std=c++11)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
This requires CMake 3.1.2 according to [Conan's documentation](http://docs.conan.io/en/latest/reference/generators/cmake.html#conan-basic-setup)

Reply:
Nice catch!